### PR TITLE
Make timeout configurable

### PR DESCRIPTION
**Pull request by EDsCODE**
Http client timeout should be configurable as the 750ms default is restrictive

### CODE REVIEW

File: lib/Client.php [hunk 1]
```diff
@@ -76,7 +76,9 @@ public function __construct(string $apiKey, array $options = [], ?HttpClient $ht
             $options['ssl'] ?? true,
             (int) ($options['maximum_backoff_duration'] ?? 10000),
             false,
-            $options["debug"] ?? false
+            $options["debug"] ?? false,
+            null,
```
Comment (liyiy) [inline lib/Client.php:80]: what's this null for

Comment (EDsCODE) [reply lib/Client.php:80]: https://github.com/PostHog/posthog-php/blob/master/lib/HttpClient.php#L49

**Review by liyiy — APPROVED** [review]
LGTM, obviously merge after tests are fixed, of course 😁

File: .github/workflows/php.yml [hunk 2]
```diff
@@ -45,3 +46,4 @@ jobs:
           files: "**.php" # you may customize glob as needed
           phpcs_path: php phpcs.phar
           standard: phpcs.xml
+          scope: "file"
```
Comment (EDsCODE) [inline .github/workflows/php.yml:49]: There's a bug that causes the default to not respect the standards xml: https://github.com/tinovyatkin/action-php-codesniffer/issues/40

**Comment by EDsCODE** [comment]
indent changes here don't matter. Leftover from trying to debug phpcs

**Pull request merged**